backend/.env

```diff
@@ -1,3 +1,3 @@
-MONGO_USER=testusr
+MONGO_USER=username
 MONGO_PASSWORD=password
 MONGO_DB=communityconnect-labs
```

Comment:
could u remove this file and put it in the gitignore + put the contents into channel as a pinned post